src/book/mdbook.rs

```diff
@@ -38,7 +38,7 @@ impl MDBook {
                         .set_src(&root.join("src"))
                         .set_dest(&root.join("book"))
                         .to_owned(),
-            renderer: Box::new(HtmlHandlebars::new()),
+            renderer: Box::new(Pandoc::new()),
```

Comment:
In the long term I guess it would be nice if the user could choose a couple of formats in his `book.json` and on build all formats would be exported. This needs a little bit of thinking to come up with a good design:

- How to handle the automatic switching of renderers
- Where to export, currently the html version is exported in `book` dir but where would pdf, epub, ... go?